pkg/user/handler.go

```diff
@@ -183,14 +192,30 @@ func (h Handler) RefreshToken(c *gin.Context) {
 	//   201: Tokens
 	//   400: Error
 	//   415: Error
-	var request RefreshTokenRequest
 
-	if err := handler.DataBinder(c, &request); err != nil {
-		_ = c.Error(err)
+	var refreshTokenString string
+
+	cookie, err := c.Cookie("refreshToken")
+	if err == nil {
+		refreshTokenString = cookie
+	}
+
+	// TODO: what if err isn't http.ErrNoCookie
+	if errors.Is(err, http.ErrNoCookie) {
```

Comment:
Checking the error type after checking whether the error is nil doesn't make sense to me. Maybe the order needs to be changed or not even check if there's no error. 🤔